taipy/core/_orchestrator/_dispatcher/_development_job_dispatcher.py

```diff
@@ -43,3 +46,9 @@ def _dispatch(self, job: Job):
         """
         rs = _TaskFunctionWrapper(job.id, job.task).execute()
         self._update_job_status(job, rs)
+
+    def _is_dispatched(self, job_id: str) -> bool:
+        return False
+
+    def _remove_job(self, job_id: str):
```

Comment:
I'd prefer `_remove_dispatched_job` to avoid confusion

Reply:
Yes, that's better.